pkgs/tools/networking/yggdrasil/0.4.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests }:
```

Comment:
Is it possible to just parametrize this file instead of copying the entire config? Also it seems like it would just be best to replace 0.3.